src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -585,6 +585,7 @@ public virtual void Clear()
 
             foreach (NodeModel el in Nodes)
             {
+                el.RaisesModificationEvents = false;
```

Comment:
Deleting connectors will trigger nodemodel's `OnModified` event, which consequently runs the graph with invalid inputs. Make nodes be silent.